servicetalk-http-router-jersey/src/main/java/io/servicetalk/http/router/jersey/DefaultJerseyStreamingHttpRouter.java

```diff
@@ -202,9 +207,23 @@ private void handle0(final HttpServiceContext serviceCtx, final StreamingHttpReq
             requestUriBuilder.append(requestTarget);
         }
 
+        final URI baseURI;
+        final URI requestURI;
+        try {
+            baseURI = URI.create(baseUri.toString());
+            requestURI = URI.create(requestUriBuilder.toString());
+        } catch (Throwable cause) {
+            Buffer message = serviceCtx.executionContext().bufferAllocator().fromAscii(cause.getMessage());
```

Comment:
Exception msg may leak some server details, which is hard to control if this `try` block changes in future. Consider logging the exception and generating a response without payload body. Alternatively, consider catching only specific exception types relevant to `URI`.

Reply:
I'll refine/narrow the exceptions type